include/CIndexStoreDB/CIndexStoreDB.h

```diff
@@ -49,6 +49,7 @@
 INDEXSTOREDB_BEGIN_DECLS
 
 typedef void *indexstoredb_object_t;
+typedef void *indexstoredb_creation_options_t;  // Not a indexstoredb_object_t since it has a separate dispose.
```

Comment:
Nitpick: I'd move this down with the others and I'm not sure the comment is really needed - it's not a `indexstoredb_object_t` because it ... isn't one. Eg. same as `indexstoredb_symbol_t`, `indexstoredb_symbol_occurrence_t`, etc.

Reply:
Done